### PR TITLE
Add Missing certificate_required on IANA change

### PR DESCRIPTION
**Pull request by alagoutte**
and fix also not needed comma

### CODE REVIEW

**Comment by Lekensteyn** [comment]
:+1: was about to fix this too.

Change was introduced by @davidben in f6f752f9af24f99241ba98510cfb1dd758bf4bf9 (included with draft 16).

File: draft-ietf-tls-tls13.md [hunk 1]
```diff
@@ -4060,12 +4060,13 @@ and their allocation policies are below:
 
 -  TLS Alert Registry: Future values are allocated via Standards
   Action {{RFC5226}}. IANA [SHALL update/has updated] this registry
-  to include values for "end_of_early_data" and "missing_extension".
+  to include values for "end_of_early_data", "missing_extension"
```
Comment (davegarrett) [inline draft-ietf-tls-tls13.md]: add a comma after "missing_extension"

File: draft-ietf-tls-tls13.md [hunk 2]
```diff
@@ -4060,12 +4060,13 @@ and their allocation policies are below:
...
+  and "certificate_required".
 
 -  TLS HandshakeType Registry: Future values are allocated via
   Standards Action {{RFC5226}}. IANA [SHALL update/has updated] this registry
   to rename item 4 from "NewSessionTicket" to "new_session_ticket"
-  and to add the "hello_retry_request", "encrypted_extensions",
```
Comment (davegarrett) [inline draft-ietf-tls-tls13.md]: Please leave the Oxford comma alone. This is correct as-is.

**Comment by alagoutte** [comment]
Hi,

I have update the change !

**Pull request merged**